### DISCUSSION TITLE
OceanBioME_example_data

### DISCUSSION THREAD

**Discussion by syou83syou83** [OceanBioME/OceanBioME.jl#27] [General]
Hi @jagoosw,

After I `git pull`, the folder OceanBioME.jl/examples/OceanBioME_example_data/ is empty except a README.md file. Do the users need to download these example data separately from rep sichen-org/OceanBioME_example_data or did I do something wrong? Thanks.

**Comment by jagoosw** [comment]
Yeah how it's setup now when you're in the examples folder you need to `got lfs clone ` the other repository

**Comment by jagoosw** [comment]
Just came across [DataDeps.jl](https://www.oxinabox.net/DataDeps.jl/stable/) which may be a better solution to this
